openedxstats/apps/sites/apps.py

```diff
@@ -3,3 +3,4 @@
 
 class SitesConfig(AppConfig):
     name = 'sites'
+    default = False
```

Comment:
https://docs.djangoproject.com/en/3.2/ref/applications/#django.apps.AppConfig.default
```
Set this attribute to False to prevent Django from selecting a configuration 
class automatically. This is useful when apps.py defines only one AppConfig 
subclass but you don’t want Django to use it by default.
```